package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -380,6 +394,16 @@ dhcp_add() {
 	config_get options "$cfg" options
 	config_get_bool dynamicdhcp "$cfg" dynamicdhcp 1
 
+	if [ $ADD_LOCAL_HOSTNAME -gt 0 ] ; then
+		local dashedname="$( echo $ifname | sed 's/\./-/g' )"
+		local routername="$( uci_get system @system[0] hostname OpenWrt )"
+		# Put the router host name on this DHCP served interface address(es)
+		# All IP6 addresses discovered by dnsmasq will be labeled (except fe80::)
+		xappend "--interface-name=$dashedname.$routername.$DOMAIN,$ifname"
+		xappend "--interface-name=$routername.$DOMAIN,$ifname"
+		xappend "--interface-name=$routername,$ifname"
+	fi
+
```

Comment:
Yeah. Until recently my own test of the other was `gateway.out` but now that I have a sensible pattern...